lib/Minimap.js

```diff
@@ -28,9 +29,12 @@ import {
 
 import { getVisual } from 'diagram-js/lib/util/GraphicsUtil';
 
+import IdGenerator from 'diagram-js/lib/util/IdGenerator';
```

Comment:
We must declare this as an external dependency in rollup config.

Reply:
![image](https://github.com/bpmn-io/diagram-js-minimap/assets/58601/e406e68a-0665-43b0-b02c-1406b220d2b6)

Reply:
added a regex to ensure this will not happen again: [4d19666](https://github.com/bpmn-io/diagram-js-minimap/pull/82/commits/4d19666be0fb231db1fc8c18fdddd30736b65b8b)